platform/ios/docs/pod-README.md

```diff
@@ -94,6 +94,6 @@ class ViewController: UIViewController {
 }
 ```
 
-Full API documentation is included in this package, within the `documentation` folder. For more details, read “[First steps with the Mapbox iOS SDK](https://www.mapbox.com/help/first-steps-ios-sdk/)” and consult the [online examples](https://www.mapbox.com/ios-sdk/examples/).
+Full API documentation is included in this package, within the `documentation` folder. For more details, read “[First steps with the Mapbox Maps SDK for iOS](https://www.mapbox.com/help/first-steps-ios-sdk/)” and consult the [online examples](https://www.mapbox.com/ios-sdk/examples/).
```

Comment:
Heads-up that the URL to [this guide](https://www.mapbox.com/help/first-steps-ios-sdk/) will likely change as soon as it gets retitled to “First steps with the Mapbox Maps SDK for iOS”.